packages/flutter/lib/src/material/navigation_rail.dart

```diff
@@ -576,18 +558,24 @@ class _RailDestination extends StatelessWidget {
 
     switch (labelType) {
       case NavigationRailLabelType.none:
-        final Widget iconPart = SizedBox(
-          width: minWidth,
-          height: minWidth,
-          child: Center(
-            child: _AddIndicator(
-              addIndicator: useIndicator,
-              indicatorColor: indicatorColor,
-              isCircular: true,
-              indicatorAnimation: destinationAnimation,
-              child: themedIcon,
+        final Widget iconPart = Column(
+          children: <Widget>[
+            if (material3) const SizedBox(height: 6),
```

Comment:
Where does this come from?

Reply:
This was a bit tricky and I should probably document it. There isn't a token for it, but there is supposed to be 12dp between the nav destinations.  In this case I needed to keep the icon centered so that it lined up with the label when the rail was extended (which is something not covered in the M3 spec). I will make this an internal constant and document it better. Thx.